### DISCUSSION TITLE
Is it possible to do training for certain classes that have a small AP without having to repeat the entire training class from zero again?

### DISCUSSION THREAD

**Discussion by kemaltf** [AlexeyAB/darknet#8417] [Q&A] [answered]
I've been training in YOLOv4, after doing the training I saw that one class has 53% AP because that class has a few label images. Can I just train those lacking classes? Or do I have to repeat the training from the beginning again?

**Comment by Tony904** [comment, marked as answer]
Unless your loss has completely plateaued I would just continue training like normal. In my experiences, yolov4 is great at preventing overfitting even for long training sessions, so the classes that are already well-trained shouldn't suffer from additional training.
